tests/napsu_mq/mst_test.py

```diff
@@ -15,12 +15,41 @@
 
 import unittest
 
+from twinify.napsu_mq.marginal_query import FullMarginalQuerySet
 
 # TODO: write tests
 
 class MSTTest(unittest.TestCase):
-    def setUp(self):
-        pass
+
```

Comment:
Shouldn't these tests be in `marginal_query_test.py`? They aren't testing anything related to MST.

Reply:
that is absolutely correct and I have fixed it now